### PR TITLE
How to Register OData resources in the Data Hub Catalog (with or without the Mendix Cloud)

### PR DESCRIPTION
**Pull request by ezagroba**

### CODE REVIEW

**Comment by ezagroba** [comment]
This is ready for review. The goal of this page is to give users a clear picture of all the ways they can register services in the Catalog, whether or not they're using the Mendix Cloud.

Some of the details about the Personal Access Tokens, Transformation API, and UI form are pared down from existing pages. Removing those source pages (or parts of pages) will be tackled in a separate merge request and with Georg's input.

**Comment by MariaShaposhnikova** [comment]
@ezagroba , where should this document sit? Should it be placed in the **Data Hub Catalog** category? Right now it's hanging nowhere and will not show up in the Guide structure.

**Comment by ezagroba** [comment]
@MariaShaposhnikova I had the same question. Does it make sense to put it under `Data Hub Guide` https://docs.mendix.com/data-hub/, at the same level as `Share Data Between Apps` https://docs.mendix.com/data-hub/share-data/ is now?

**Comment by MariaShaposhnikova** [comment]
@ezagroba , it seems to be more relevant to the [Data Hub Catalog](https://docs.mendix.com/data-hub/data-hub-catalog/), I can place it after [Register Data Assets]( https://docs.mendix.com/data-hub/data-hub-catalog/register) on the same level.

**Comment by MariaShaposhnikova** [comment]
@ezagroba , there are links to now non-existing **Manually Registering a Data Source from Enterprise Applications** section that used to be in **Register Data Assets** and now the build gives warnings. This section is usually linked in the context of OData v4 services, shall I leave this context and just update the link?

**Comment by ezagroba** [comment]
@MariaShaposhnikova Yes please.

File: content/releasenotes/data-hub/index.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ These release notes cover changes made to the [Mendix Data Hub](/data-hub/).
 
 #### New Features
 
-* A new connector has been added to the **Data Hub Home** for [registering](/data-hub/data-hub-catalog/register#registration-form) Siemens Opcenter data sources.
+* A new connector has been added to the **Data Hub Home** for [registering](/data-hub/data-hub-catalog/register-data) Siemens Opcenter data sources.
```
Comment (MariaShaposhnikova) [inline content/releasenotes/data-hub/index.md:64]: @ezagroba , I changed the link as discussed, but I have doubt whether it's correct now. We do not seem to mentioned DH Home in the new doc.

File: content/releasenotes/data-hub/index.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ These release notes cover changes made to the [Mendix Data Hub](/data-hub/).
 #### Improvements
 
 * Datasets in the [search results list](/data-hub/data-hub-catalog/search#search-results) that are validated are now clearly indicated as such.
-* Custom owners can be added for an application when [registering data sources](/data-hub/data-hub-catalog/register#registration-form) using the business application connectors from the **Data Hub Home**.
+* Custom owners can be added for an application when [registering data sources](/data-hub/data-hub-catalog/register-data) using the business application connectors from the **Data Hub Home**.
```
Comment (MariaShaposhnikova) [inline content/releasenotes/data-hub/index.md:76]: Same with this link, is the document we have now the right document to refer to here?

File: content/releasenotes/data-hub/index.md [hunk 3]
```diff
@@ -98,13 +98,13 @@ The [Data Hub Transform API](https://datahub-spec.s3.eu-central-1.amazonaws.com/
 
 #### Improvements
 
-*  [Registration](/data-hub/data-hub-catalog/register#registration-form) of data sources using the business application connectors from the **Data Hub Home** is improved. For each connector, users can register data sources by selecting from a list of apps and environments already registered in the Catalog for the business application. When none are registered, the  "**Create New...**" option will be presented.
+*  [Registration](/data-hub/data-hub-catalog/register-data) of data sources using the business application connectors from the **Data Hub Home** is improved. For each connector, users can register data sources by selecting from a list of apps and environments already registered in the Catalog for the business application. When none are registered, the  "**Create New...**" option will be presented.
```
Comment (MariaShaposhnikova) [inline content/releasenotes/data-hub/index.md:101]: In the new document, we do not mention business application connectors, were they removed? In this case I will delete links. This concern applies to all links I've changed in this document.

**Comment by MariaShaposhnikova** [comment]
I can't add this comment in-line, the **register-data** doc has links to #transform-api but this achor does not exist. Where should these links go?

File: content/releasenotes/data-hub/index.md (hunk 1 above)
Comment (ezagroba) [reply content/releasenotes/data-hub/index.md:64]: @MariaShaposhnikova I expect it is correct, the new page you're referencing is part of this merge request.

File: content/releasenotes/data-hub/index.md (hunk 2 above)
Comment (ezagroba) [reply content/releasenotes/data-hub/index.md:76]: Yes.

File: content/releasenotes/data-hub/index.md (hunk 3 above)
Comment (ezagroba) [reply content/releasenotes/data-hub/index.md:101]: The `business application connectors` are linked from buttons on the homepage. They're still there. 

It's still accurate.

**Comment by ezagroba** [comment]
@MariaShaposhnikova I updated the section header and anchor link from `Transformation` to `Transform`, good catch.

**Comment by MariaShaposhnikova** [comment]
@ezagroba, mine review is mostly ready (I'll go through the document one final time), but are we waiting for the input from someone from the team or is this good to be merged?

**Comment by ezagroba** [comment]
@MariaShaposhnikova No one from the team has provided input. Let's go ahead and merge it.

**Pull request merged**